include/deal.II/matrix_free/shape_info.h

```diff
@@ -196,14 +208,14 @@ namespace internal
       /**
        * Stores the shape gradients of the shape function space associated to
        * the quadrature (collocation), given by FE_DGQ<1>(Quadrature<1>). For
-       * faster evaluation only the even-odd format is necessary.
+       * faster evaluation it is stored in the even-odd format.
```

Comment:
Maybe slightly reword this now that we have the same setup as with `shape_values`. How about writing the second sentence as "This array provides an alternative representation of the `shape_gradients_collocation` field in the even-odd format."